src/WebJobs.Script.Host/Program.cs

```diff
@@ -19,11 +23,38 @@ static void Main(string[] args)
 
             ScriptHostConfiguration config = new ScriptHostConfiguration()
             {
-                RootScriptPath = rootPath
+                RootScriptPath = rootPath,
+                FileLoggingEnabled = true
             };
 
+            // For local execution, allow reading secrets from a file. 
```

Comment:
Not sure about this. For local execution, the already established pattern is to put your settings in env vars. That's how my local dev box is configured - works great. Recommend using that rather than introducing another pattern.